ci/properties/symfony.properties.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Symfony",
+    "description": "Test a Symfony project.",
+    "iconName": "php",
+    "categories": [
```

Comment:
Can you add `Continuous integration` as a category value in the first line like the one mentioned [here](https://github.com/actions/starter-workflows/blob/c36ea2c5605bf159b7a6ca8b8bc97c410accc036/ci/properties/laravel.properties.json#L6)?

FYI, this is going to be added as a [validation](https://github.com/actions/starter-workflows/pull/1099) soon as it would help in better grouping and recommendation of templates.